.scripts/e2e.sh

```diff
@@ -5,12 +5,13 @@
 # -d (database) target database for expectations
 # -a (auth) optional credentials for database target
 
-while getopts 'b:d:a:' v
+while getopts 'b:d:a:m:' v
 do
   case $v in
     b) SQL_RUNNER_PATH=$OPTARG ;;
     d) DATABASE=$OPTARG ;;
     a) CREDENTIALS=$OPTARG ;;
+		m) MODEL=$OPTARG ;;
```

Comment:
Indent has gone a bit wonky here and in a few other places. Check your indentation settings in your text editor - it's either the setting to use tabs/spaces, or the one to specify how many indents to use, one of them (can't remember which) doesn't play nice with GH.